### PR TITLE
docs: include UI specs in README

### PR DESCRIPTION
**Pull request by obdulia-losantos**
:books: :books: :books: :books:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +100,7 @@ of the most common ones with non default values. For more info take a look at th
 - `AETHER_KERNEL_URL_TEST`: `http://kernel-test:9000` Aether Kernel Testing Server url.
 - `AETHER_ODK_TOKEN`: `d5184a044bb5acff89a76ec4e67d0fcddd5cd3a1`
   to connect to it from other modules. It's used within the start up scripts.
+- `ADMIN_PASSWORD`: `adminadmin`. It's used within the start up scripts.
```
Comment (shawnsarwar) [inline README.md:103]: @quesebifurcan ; I thought we were changing this?

**Review by shawnsarwar — COMMENTED** [review]
I'm not sure where we landed with default passwords. I think Fredrik was working on something?

File: README.md (hunk 1 above)
Comment (obdulia-losantos) [reply README.md:103]: Yeah! He was taking out the `/conf/extras/initial.json` files, but the start up script will use these env vars. 

We are not using it locally but we need to inform the user or the container will never start :thinking: 

https://github.com/eHealthAfrica/aether/pull/252

Comment (quesebifurcan) [reply README.md:103]: What Obdulia said -- some modules still have hardcoded credentials which would need more refactoring than we have time for before Friday. #252 will make sure that no development passwords will be included in the images.

Comment (obdulia-losantos) [reply README.md:103]: Another mandatory variable is `DJANGO_SECRET_KEY`, without it the Django app does not start.

**Pull request closed without merging**